### DISCUSSION TITLE
Cloned keys but different plots for 2 PC

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#5801] [Q&A]
Let's assume I have 2 PC in 2 locations.

Each PC has cloned .chia keys. Each PC runs full node, farming, wallets with the same keys.

Difference is only in plots. Each PC has own set of plots.

Will such setup work? I mean proof will be confirmed only by some single PC (which has corresponding plot).

If not which options do we have?
VPN? Or 2 different wallets?
Or only absolutely different all keys (including plotting keys)?

**Comment by ghost** [comment]
Guys any ideas?

**Reply by Tofandel** [reply]
Of course it will work, as long as the plots are not the same, doesn't matter if you use the same key on another machine

**Reply by ghost** [reply]
> 
> 
> Of course it will work, as long as the plots are not the same, doesn't matter if you use the same key on another machine

Thanks!

Regarding keys... If they are different how rewards will be sent to original wallet? Is reward related only to public key used during plotting?

**Reply by Tofandel** [reply]
Yes, the reward is sent to the public key of the winning plot, so if your plot was generated using a different key, it will be sent to that key. That also means that if your farmer doesn't have the private key of the plot, it will not be able to farm
